### DISCUSSION TITLE
Как откатиться к 1.4.1?

### DISCUSSION THREAD

**Discussion by Valun4e** [ilyhalight/voice-over-translation#460] [Q&A] [answered]
Новейшая версия безбожно глючит. Старая моментально переводила, а сейчас для перевода трехминутного(!) видео предлагает ждать минуту. Более чем получасовые видео не переводит вообще, показывая ошибку. Да и интерфейс испортился. Прошу Вас, уважаемый автор, подскажите, как мне вернуться на прекраснейшую версию 1.4.1?

**Comment by Valun4e** [comment]
Блин, ну дёрнул чёрт меня обновиться... Ведь старая версия(1.4.1) работала просто великолепно, безукоризненно! Какой же я идиот! Автор(ы) прекрасного скрипта, пожалуйста, подскажите как мне вернуть старую? И желательно нетрудным способом. А то я плохо соображаю - была тяжелая травма головного мозга при ДТП. Заранее спасибо!

**Comment by ilyhalight** [comment, marked as answer]
Написал инструкцию в комментариях к 1.5.0

https://github.com/ilyhalight/voice-over-translation/discussions/459#discussioncomment-7935819
